app/services/sign_in/constants/auth.rb

```diff
@@ -3,9 +3,22 @@
 module SignIn
   module Constants
     module Auth
+      [ONE = 1, TWO = 2, THREE = 3].freeze
```

Comment:
I mentioned above to make these `IAL_ONE`, `IAL_TWO` (or something like that). We'll also need to make this `LOA_THREE`